### DISCUSSION TITLE
MCDU server and Simbridge collide

### DISCUSSION THREAD

**Discussion by MStallen** [flybywiresim/aircraft#7367] [Help]
If I enable Simbridge then the MCDU server doesn't start up. Port already in use. How to prevent that?

**Comment by Lucky38i** [comment]
Hi, you can modify the settings from simbridge to use a different port or find the properties.json in the simbridge folder wherever it may be installed.

However to be frank, we have not announced Simbridge and it's usage to the addon as of yet so I would refrain from using it until we announce it and the required documentation for it.
